### ISSUE TITLE
Dataset posted 3/23/20 is incorrect

### ISSUE THREAD

**Issue by kirtu** [CSSEGISandData/COVID-19#1337]
The time_series_19-covid-Confirmed.csv file is incorrect.  The values for 3/23/20 are duplicates of the values for 3/22/20.

**Comment by chyland3** [comment]
This is true for most rows, but random rows did get updated. It appears about 10% of rows were updated and the rest simply got copied over.

**Comment by cvelas31** [comment]
This problem also happens in the time_series_19-covid-Deaths.csv

**Comment by dejan-shuki** [comment]
Confirmed is probably either Deaths or Recovered.

**Comment by dejan-shuki** [comment]
When you click on file you will see the notice that it is deprecated.
We should use Global files
